diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/comments/HeaderCommentRule.kt

```diff
@@ -135,13 +135,13 @@ class HeaderCommentRule(configRules: List<RulesConfig>) : DiktatRule(
             return
         }
 
-        if (makeCopyrightCorrectYear(configuration.getCopyrightText()).isNotEmpty()) {
-            log.warn("Copyright year is not up do date.")
-        }
-
         // need to make sure that copyright year is consistent with current year
         val copyrightText = configuration.getCopyrightText()
 
+        if (makeCopyrightCorrectYear(copyrightText).isNotEmpty()) {
```

Comment:
Please add this to the readme of the rule

Reply:
done